backend/lib/components/Auth0.js

```diff
@@ -1,47 +1,69 @@
 const axios = require("axios");
-const {
-  config: { auth },
-} = require("../../config");
-const User = require("./User");
+const qs = require("querystring");
+const { config } = require("../../config");
 
-const oauth = async () => {
```

Comment:
Why did you remove oauth?

Reply:
Hmm don't think I removed this on purpose, maybe merge mistake.
But if I look at your branch it wasn't there also? https://github.com/manualzuru/FightPandemics.com/blob/dac1404bc9aa1d1b1af4af1837b2ad5cd0461f5e/backend/lib/components/Auth0.js

The `/api/auth/oauth` route only calls  `Auth0.authenticate()`

Reply:
The oauth is the most important function in this PR, you shoudn't touch something you don't know without communicating before. This is totally disrespect and only shows me the lack of professionalism you have

Reply:
@neemiasjnr again, I did not remove this on purpose,. I need to double check what's going on.
Also, this is a pull request, it hasn't even been merged yet, it seems like you are trying to turn things around, it's not needed to call me out like this since there is absolutely no ground to accuse me like this.

Reply:
See this was renamed to 
` const authenticate = async (grantType, payload = {}) => { 
...
eturn axios
    .post(`${config.auth.domain}/oauth/token`, qs.stringify(body), {
```
which is making the same request to Auth0